nav2_waypoint_follower/include/nav2_waypoint_follower/waypoint_follower.hpp

```diff
@@ -108,8 +108,7 @@ class WaypointFollower : public nav2_util::LifecycleNode
    * @brief Action client goal response callback
    * @param future Shared future to goalhandle
```

Comment:
Hi Sarthak, Can you also update the comments on line 109 so it matches the code (future -> goal)?

Reply:
Ah, yes missed this!